decidim-verifications/spec/events/decidim/verifications/managed_user_error_event_spec.rb

```diff
@@ -22,7 +23,7 @@
 
   describe "resource_url" do
     it "is generated correctly" do
-      expect(subject.resource_url).to eq("http://#{resource.current_user.organization.host}/profiles/#{resource.current_user.nickname}")
+      expect(subject.resource_url).to eq("http://#{organization_host}/profiles/#{resource.current_user.nickname}")
```

Comment:
If we fix the port for the URLs, these expectations need to also assume the port is there e.g. similarly to:
https://github.com/decidim/decidim/blob/8664823a2370ce994406f777de11fb543ad429be/decidim-assemblies/spec/presenters/decidim/assemblies/assembly_presenter_spec.rb#L29

Reply:
Preferably the fix would apply to all these URLs, as the other introduced URLs will still be without the port and correct protocol depending on the configuration.

I actually thought that you'd wrap the engine router to its own method, as in:
```ruby
      def decidim_admin
        @decidim_admin ||= Decidim::EngineRouter.new("decidim_admin", { host: managed_user.organization.host })
      end
```

Then I believe those methods already delegated to this would work correctly (?).